### DISCUSSION TITLE
You can fix ds and dsi wifi

### DISCUSSION THREAD

**Discussion by ghost** [rafaelvcaetano/melonDS-android#749] [Ideas]
All you have to do to implement wifi is Merge libslirp from pc version of melonds
And add ssl support

**Comment by rafaelvcaetano** [comment]
Sounds good, doesn't work. libslirp relies on low level access to sockets, which is not available on Android. I'm still trying to figure out the best way to add proper support.

**Comment by ghost** [comment]
Can you implement it for rooted devices?

**Comment by ghost** [comment]
I have a rooted device so please inplement that low level acesss for rooted devices?
